js/wdi_radio/songs.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+(function(){
+  angular
+  .module("wdiRadio", [
+    "ngResource"
+  ])
+  .config(function($sceProvider){
+    $sceProvider.enabled(false);
```

Comment:
Nice job here! I have used $sce in a different way, so it's nice to see a new approach to get the songs to play